cli/main.py

```diff
@@ -130,7 +124,7 @@ def get_settings_files(self):
         settings_files = []
 
         # first take all input files from args
-        for input_file in self.args['input'] or []:
+        for input_file in self.args.get('input', []) or []:
```

Comment:
Either `args.get('input') or []` or `args.get('input', [])`

Do we expect `args["input"]` to hold something other than a list?